7.9.вложенные_циклы_часть_2/7.9.1.py

```diff
@@ -0,0 +1,7 @@
+num = int(input())             
+counter = 0                      
+for y in range(1, num + 1):    
+    for x in range(y):         
```

Comment:
`x` не используешь, значит `_`:
```suggestion
    for _ in range(y):         
```